classes/submission/maps/Schema.php

```diff
@@ -1191,4 +1198,12 @@ protected function getStageFilesBySubmissions(Enumerable $submissions, array $st
             ->filterByFileStages($stageIds)
             ->getMany();
     }
+
+    /**
+     * Implement by a child class to populate app specific data.
```

Comment:
I think it's interesting to add a `@param` for the `$submissions`, then future code analyzers will be able to understand what's going on.